src/main/java/competition/subsystems/drive/commands/SwerveDriveWithJoysticksCommand.java

```diff
@@ -274,4 +278,16 @@ private double scaleHumanRotationInput(double humanInputPower) {
         }
         return humanInputPower;
     }
+
+    /**
```

Comment:
it feels like this should go somewhere more common so that we don't have to copy and paste it for any drive with joysticks logic?